exercises/e001/results/mikjarz/e001.py

```diff
@@ -0,0 +1,33 @@
+import random
+import string
+import base64
+import argparse
+
+parser = argparse.ArgumentParser(description="Random string of base32 characters")
+parser.add_argument('-a', '--amount', type=int, help='How many characters to print')
```

Comment:
optional argument, see other pull requests, in short `cmd.py -a 12` vs `cmd.py 12`